### DISCUSSION TITLE
能麻烦添加下AX210无线网卡驱动吗

### DISCUSSION THREAD

**Discussion by qwer99630** [istoreos/istoreos#673] [General]
装了个AX210无线网卡，提示
[   55.922914] iwlwifi 0000:06:00.0: no suitable firmware found!
[   55.922972] iwlwifi 0000:06:00.0: minimum version required: iwlwifi-ty-a0-gf-a0-39
[   55.923047] iwlwifi 0000:06:00.0: maximum version supported: iwlwifi-ty-a0-gf-a0-59
[   55.923113] iwlwifi 0000:06:00.0: check git://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git
求大佬添加个AX210驱动

**Comment by jjm2473** [comment]
ok，但是不知道能不能用。
你先自己测试没问题，我再集成到固件。

测试方法：
1. 下载 https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/plain/iwlwifi-ty-a0-gf-a0-59.ucode
2. 将下载到的 `iwlwifi-ty-a0-gf-a0-59.ucode` 放在设备的 `/lib/firmware/` 文件夹下，注意文件名不能改
3. 重启设备
4. 如果内核日志没问题了，执行 `wifi config` 自动生成配置，之后luci界面应该可以看到无线菜单


注意：iwlwifi不支持AP模式，如果遇到任何问题，先在luci删掉AP配置

**Comment by jjm2473** [comment]
x86已经支持

**Reply by mrgaolei** [reply]
请问现在可以AX210在AP模式了吗？

**Reply by jjm2473** [reply]
intel的部分型号可以开启2.4G AP，但都不支持5G AP https://wireless.wiki.kernel.org/en/users/drivers/iwlwifi 。
总而言之就是没什么卵用
